### PR TITLE
Use black style

### PR DESCRIPTION
**Pull request by eemeli**
This applies and enforces code styling by black in the project.

The PR includes the addition of a `.git-blame-ignore-revs` file that allows for ignoring commit 0efa923 in `git blame` output. To use it, run the following in the repo root to update your local git config:

    git config blame.ignoreRevsFile .git-blame-ignore-revs

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -13,17 +13,33 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest]
-        python-version: ['3.7', '3.8', '3.9']
+        python-version: ["3.7", "3.8", "3.9"]
```
Comment (flodolo) [inline .github/workflows/test.yaml:16]: Should we add 3.10 here?

File: setup.py [hunk 2]
```diff
@@ -25,35 +25,34 @@
...
-      entry_points={
-        'console_scripts':
-        [
-            'compare-locales = compare_locales.commands:CompareLocales.call',
-            'moz-l10n-lint = compare_locales.lint.cli:main',
+setup(
+    name="compare-locales",
+    version=version,
+    author="Axel Hecht",
+    author_email="axel@mozilla.com",
```
Comment (flodolo) [inline setup.py:32]: Should we change these, given the email is not valid anymore and you're maintaining this project?

**Review by flodolo — APPROVED** [review]
A few questions/notes not really related to this PR (which is OK to merge) caused by seeing ancient code.

File: setup.py (hunk 2 above)
Comment (eemeli) [reply setup.py:32]: Probably, yes. I'll do so in a separate PR, though.

**Pull request merged**